### ISSUE TITLE
Nodes not registering with Hub

### ISSUE THREAD

**Issue by asdutoit** [mendhak/selenium-grid-ecs#4]
Hi There

I followed your tutorial on [Mendhak](https://code.mendhak.com/selenium-grid-ecs/) and for the most part, everything seems to work great except for one thing.  
It looks like the Nodes (Chrome and Firefox) doesnt register with the Hub. 
I followed the instructions to a T but for the life of me cannot figure out what I am doing wrong. 
<img width="663" alt="Screenshot 2022-02-03 at 15 48 03" src="https://user-images.githubusercontent.com/20240370/152355243-4d682f96-785d-4ef1-b87c-50b9e10f06d1.png">

I see the following in the logs and suspect this is the culprit:

```
SE_EVENT_BUS_HOST not set, exiting!
```

Do you perhaps have any pointers?  Or could it be that some code may have been updated since you created the tutorial?

**Comment by mendhak** [comment]
Yes it's quite possible the image has been updated since the tutorial, it's happened to me a few times in fact where they've changed what the image is doing, but they kept the `3.14159` tag intact.  

Could you try adding a new environment variable here:  https://github.com/mendhak/selenium-grid-ecs/blob/master/main.tf#L292

Set `SE_EVENT_BUS_HOST` to `hub.selenium`.  

If that does work for you, I think set the Docker image in your Terraform to `selenium/hub:3.141.59-20210929` so it should stay fixed.  

If that doesn't work for you, try one tag earlier, which is `selenium/hub:3.141.59-20210913`

**Comment by asdutoit** [comment]
Hi there, 

Sorry for the late reply.  I will do so and revert. 

Thank you for the quick response.  Appreciate your willingness to assist.
